WordPress/Classes/ViewRelated/Activity/ActivityListViewController.swift

```diff
@@ -43,6 +43,9 @@ class ActivityListViewController: UITableViewController, ImmuTablePresenter {
             self?.refreshModel()
         }
 
+        refreshControl = UIRefreshControl()
+        refreshControl?.addTarget(self, action: #selector(ActivityListViewController.userRefresh), for: .valueChanged)
```

Comment:
I am not a big fan of specifying the class name, when passing a selector. I'm asking to understand, not necessarily to trigger a discussion: what would be the advantage of specifying the class name? Isn't it implicit, if not specified, that the selector would be in the defined in the current context?

Reply:
Honestly, mostly because I was looking at how it was implemented in Plugins. No idea why I included it there though!